.github/jira-mapping-rules.yml

```diff
@@ -0,0 +1,11 @@
+defaultJiraIssueType: 10009
+rules:
+  - type: label
+    labels:
+      - bug
+    jiraIssueType: 10011
+  - type: title
+    searchTerms:
+      - feature
```

Comment:
Ok, so we could take some minor control e.g. change this to - "[feature]" to avoid clashes - and that would be fine?